components/homme/src/test_src/dcmip12_wrapper.F90

```diff
@@ -634,7 +634,7 @@ subroutine set_hybrid_coefficients(hv, hybrid, eta_t, c)
     tmp        = max( (hv%etai(k)-eta_c)/(1.0-eta_c), 0.0_rl)
     hv%hybi(k) = tmp**c
     hv%hyai(k) = hv%etai(k) - hv%hybi(k)
-    if(hybrid%par%masterproc) write(*,'(i4,a,f18.15,a,f18.15,a,f18.15)') &
+    if(hybrid%masterthread) write(*,'(i4,a,f18.15,a,f18.15,a,f18.15)') &
```

Comment:
Why change masterproc to masterthread here? Without looking, i'd think this is not intended?

Reply:
it's in a threaded region, but we only want one thread to print this message.